openjdk/build.xml

```diff
@@ -197,40 +197,40 @@
 					<else>
 						<if>
 							<and>
-								<contains string="${env.SPEC}" substring="aarch64" />
+								<contains string="${env.SPEC}" substring="arm" />
 								<equals arg1="${JDK_VERSION}" arg2="8" />
 							</and>
 							<then>
-								<property name="jdkName" value="openjdk-aarch64-jdk8u" />
+								<property name="jdkName" value="aarch32-jdk8u" />
```

Comment:
I think https://github.com/AdoptOpenJDK/openjdk-aarch64-jdk8u and https://github.com/AdoptOpenJDK/openjdk-aarch32-jdk8u are different repos? Should https://github.com/AdoptOpenJDK/openjdk-aarch64-jdk8u be updated to https://github.com/adoptium/aarch64-jdk8u ( though I checked https://github.com/adoptium/aarch64-jdk8u wasn't setup)? Is this expected?

Reply:
Never mind. Just saw the comment in the commit https://github.com/adoptium/aqa-tests/pull/2653#issue-672456426

Reply:
To your first question: Yup, those are different repos. 

I was told that aarch64-specific code had been folded into the main openjdk src repos, so I didn't see the point of specifically redirecting to the aarch64 variant repo anymore.

aarch32, however, does have a repo at adoptium (https://github.com/adoptium/aarch32-jdk8u), and I'm told it's still needed for aarch32 support.

These reasons are why I swapped the aarch64 redirect for an aarch32 redirect.

I hope that makes sense?